core/core_modules/help.js

```diff
@@ -1,8 +1,10 @@
 var figlet = require.safe('figlet'),
 
-constructHelpMessage = function(help, modules, context, prefix) {
-	for (var i = 0; i < modules.length; i++) {
-		var cmdHelp = modules[i].help.call(context, prefix);
+constructHelpMessage = function (help, modules, context) {
+    for (var i = 0; i < modules.length; i++) {
+        var cmdHelp = modules[i].ignoreHelpContext
+            ? modules[i].help(context.commandPrefix)
+            : modules[i].help.call(context, context.commandPrefix);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [':' should be placed at the end of the line.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530333283/issues/source?bid=3282987&fileBranchId=3282997#l7)